### PR TITLE
add service account permission list widget

### PR DESCRIPTION
**Pull request by jacob-dowdy**
add GCP service account permission list widget and layout

### CODE REVIEW

**Review by ericanagle1 — APPROVED** [review]

**Review by a-u-h-g — CHANGES_REQUESTED** [review]
Can you also add an updated screenshot?

**Review by a-u-h-g — DISMISSED** [review]

**Comment by jacob-dowdy** [comment]
Added updated board pic and widget layout

File: src/boards/gcp-iam/board.json [hunk 1]
```diff
@@ -229,267 +229,162 @@
...
-				"i": "d3d3c430-654d-40e7-a9d7-d54607d29cac",
-				"moved": false,
-				"static": false
-			},
-			{
-				"w": 6,
-				"h": 2,
-				"x": 0,
-				"y": 26,
-				"i": "18b673fb-c504-4aa8-9073-9a21997dda8e",
```
Comment (erkangz) [inline src/boards/gcp-iam/board.json:489]: Are these intentionally deleted?

File: src/boards/gcp-iam/board.json [hunk 2]
```diff
@@ -142,33 +142,33 @@
...
-			"id": "0f5a541b-fdaa-4aa1-b92e-3bad90db9536",
+			"id": "06d184f3-4f64-44ab-8748-3db3a7bb87f7",
 			"title": "Service accounts with write privileges",
 			"type": "graph",
 			"config": {
 				"queries": [
 					{
 						"name": "query1",
-						"query": "Find google_iam_service_account as user\n  that assigned google_iam_binding\n  that uses google_iam_role with readonly!=true\nreturn tree"
+						"query": "Find google_iam_service_account with tag.AccountName = 'sysdig-development-2' as user\n  that assigned google_iam_binding\n  that uses google_iam_role with readonly!=true\nreturn tree"
```
Comment (erkangz) [inline src/boards/gcp-iam/board.json:152]: is this `with tag.AccountName = 'sysdig-development-2'` hardcoded filter intentional?

File: src/boards/gcp-iam/board.json [hunk 3]
```diff
@@ -142,33 +142,33 @@
...
-			"id": "1a27a908-9ed9-4a3a-9578-7eac98308738",
+			"id": "df62a65d-6e77-4320-ba60-3a3564594d9f",
 			"title": "External users with write privileges",
 			"type": "graph",
 			"config": {
 				"queries": [
 					{
 						"name": "query1",
-						"query": "Find google_user with _source = 'system-mapper'\n  that assigned google_iam_binding\n  that uses google_iam_role with readonly!=true \n  return tree"
+						"query": "Find google_user with _source = 'system-mapper'\n  that assigned google_iam_binding with tag.AccountName = 'datadog-dev-335212'\n  that uses google_iam_role with readonly!=true \n  return tree"
```
Comment (erkangz) [inline src/boards/gcp-iam/board.json:165]: Same here - AccountName filter intentional?

**Review by erkangz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**